recipes/labgrid/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "labgrid" %}
+{% set version = "25.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/labgrid-{{ version }}.tar.gz
+  sha256: 880aea4822601f2b13b7a95ba5f3f2b7f8efca8bd55318f29aad6da99deadbaf
+
+build:
+  skip: true  # [not linux]
```

Comment:
Looks like this can be noarch.

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.